datasets/modeled/buildings/dataset.json5

```
@@ -82,7 +84,7 @@
       type: "subsector",
       class: "Subsector",
       name: "unspecified_subsector",
-      description: "Unspecified subsectors",
+      description: "Unspecified Subsectors",
```

```suggestion
      description: "Unspecified subsectors",
```

nit: Description doesn't need to be title case.